pkg/plugins/resources/k8s/native/config/rbac/role.yaml

```diff
@@ -46,6 +46,19 @@ rules:
   - patch
   - update
   - watch
+- apiGroups:
+  - kuma.io
+  resources:
+  - gatewayroutes
+  - gateways
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
```

Comment:
ISTR that the RBAC controller is OK with unknown API resources, so this ought to be OK when gateway types are not installed. It would be cleaner if we didn't install them, but that would end up with more build complexity.

WDYT?

Reply:
Yeah I think it's an OK trade off. There's still one problem though, that installing the chart via Helm leads to the new CRDs being installed, and we can't exclude them via templates.

How about adding a step to `tools/releases/helm.sh` to delete those 2 CRDs while packaging? Not ideal perhaps. WDYT @jakubdyszkiewicz

Reply:
The only clean-ish way I can see to avoid installing the gateway CRDs here is to keep them in a different source path. That would mean adding some sort of parallel helper for `deployments.KumaChartFS()` - seems doable.

We could merge this and tackle it in a followup PR if there's no risk of getting caught up in a release tag by accident.

Reply:
OK, merging as is.